src/frontend/src/lib/components/swap/Swap.svelte

```diff
@@ -36,21 +37,57 @@
 		store: icTokenFeeStore
 	});
 
-	const onOpenSwap = async (tokenId: symbol) => {
+	const isDisabled = () => isNullish($kongSwapTokensStore);
```

Comment:
- nitpick: `isDisabled = (): boolean =>`
- should we differenciate `undefined` and `null` or in both of its usage checking for both is correct?